### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by mikepmanning**
Added in url update for automatic updates

Other general fixes - couple issues, misisng tinderbox, spacing for docs, ndb fixes.

### CODE REVIEW

**Comment by mikepmanning** [comment]
@vadi2 @keneanung - will you guys please take a look at the last two commits (url change and the script for updating). See if I missed anything in the update procedure. I think everything's correct, but I want to be really sure before merging.

Thanks

File: m&m (import me).xml [hunk 1]
```diff
@@ -55525,7 +55525,7 @@ function mm.checkforupdate(type)
     end
 
     mm.checkingupdates = true
-    downloadFile(mm.versionfile, &quot;http://mm.vadisystems.com/current_version&quot;)
+    downloadFile(mm.versionfile, &quot;http://m-mf.github.io/current_version&quot;)
```
Comment (keneanung) [inline m&m (import me).xml:55528]: URL should be http://m-mf.github.io/release/current_version here

File: m&m (import me).xml [hunk 2]
```diff
@@ -55554,8 +55554,8 @@ end
 
 -- downloads the system &amp; updates the system version saved
 function mm.downloadnewsystem(newversion)
-  mm.downloadedsystem = downloadfolder..string.format(&quot;%s m&amp;m.zip&quot;, mm.me.name)
-  local downloadurl = [[http://vadisystems.com/mm-r/r/]]..mm.me.name..[[ m&amp;m.zip]]
+  mm.downloadedsystem = downloadfolder..string.format(&quot;%s m&amp;m.zip&quot;, mm.me.class)
+  local downloadurl = [[http://m-mf.github.io/m-mf/r/]]..mm.me.class..[[ m&amp;m.zip]]
```
Comment (keneanung) [inline m&m (import me).xml:55558]: URL should be [[http://m-mf.github.io/m-mf/release/]]..mm.me.class..[[ m&amp;m.zip]] here

**Comment by keneanung** [comment]
Oh... I see the changes in the second commit... This github change confuses the hell out of me...

Nevertheless it's `release` instead of `releases`

**Comment by mikepmanning** [comment]
ok, fixed that, going to close and re-open another one

**Comment by mikepmanning** [comment]
actually, looks like I don't need to, pushed the url change

**Comment by mikepmanning** [comment]
Merged this - I think I'm going to need to do two releases to test it out.

one to load in the new updates and one to make sure it's actually updating.

**Pull request merged**